apps/firefox/firefox.win.talon

```diff
@@ -27,7 +27,12 @@ action(browser.bookmarks_bar):
 
 action(browser.focus_address): 
 	key(ctrl-l)
-	
+
+tab search <user.text>:
```

Comment:
for Firefox-specific commands, I'd recommend creating a new firefox.talon file that matches on
`app: firefox`
so we don't have to maintain the commands in multiple places